### ISSUE TITLE
noise in vectors

### ISSUE THREAD

**Issue by jepetko** [openlayers/ol3-cesium#165]
There is a noise in vectors, for instance in this example (screenshots attached):
http://openlayers.org/ol3-cesium/examples/vectors.html
or
http://openlayers.org/ol3-cesium/examples/selection.html

window.navigator.appVersion returns:
```
"5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.91 Safari/537.36"
```
![vector_noise_1](https://cloud.githubusercontent.com/assets/1812641/5945247/6006df2c-a72d-11e4-933a-18fcf938fcb9.png)
![vector_noise_2](https://cloud.githubusercontent.com/assets/1812641/5945248/600aa620-a72d-11e4-800a-5b2ff3da3bfe.png)

**Comment by ahocevar** [comment]
This is because your vector data has lower elevation than the terrain model in some places. As a workaround, you can either configure your Cesium without a terrain provider, or give your vector features a higher elevation.

In the future, Cesium will support clamping of vectors to the ground, then you won't need such workarounds. See #12.

**Comment by jepetko** [comment]
thank you for the feedback.

**Comment by gberaudo** [comment]
I have the same issue on my machine; there is no link with elevation.

Could you check using sandcastle as described in this comment:
https://github.com/AnalyticalGraphicsInc/cesium/issues/2129#issuecomment-71992434

If you are impacted, please leave a comment there with you hardware
information.

**Comment by gberaudo** [comment]
A Chrome developer found the issue and the fix will be in Cesium 1.6.
@jepetko, thank you for your report.
